doc/releasenotes/8_0_0_release_notes.md

```diff
@@ -0,0 +1,16 @@
+<<<<<<< HEAD
```

Comment:
Not sure how you ended up with a merge conflict, but this needs to be resolved.

Reply:
Should be fixed?